### PR TITLE
Code Formatting Units V2

### PR DESCRIPTION
**Pull request by MrRowey**

### CODE REVIEW

File: units/UEA0104/UEA0104_Script.lua [hunk 1]
```diff
@@ -36,24 +47,30 @@ UEA0104 = ClassUnit(AirTransport) {
...
     EngineRotateBones = {'Front_Right_Engine', 'Front_Left_Engine', 'Back_Left_Engine', 'Back_Right_Engine', },
 
+    ---@param self UEA0104
+    ---@param builder Unit
+    ---@param layer string
     OnStopBeingBuilt = function(self,builder,layer)
         AirTransport.OnStopBeingBuilt(self,builder,layer)
+        local trash = self.Trash
+        local bp = self.Blueprint
+        local landingAnimManip = CreateAnimator(self)
```
Comment (clyfordv) [inline units/UEA0104/UEA0104_Script.lua:57]: This needs to be assigned to `self.landingAnimManip` or else we lose it and throw errors later.

File: units/UEA0107/UEA0107_Script.lua [hunk 2]
```diff
@@ -30,21 +40,26 @@ UEA0107 = ClassUnit(AirTransport) {
 
         PlayDestructionEffects = true,
 
+        ---@param self UEA0107
+        ---@param builder Unit
+        ---@param layer string
         OnStopBeingBuilt = function(self,builder,layer)
             AirTransport.OnStopBeingBuilt(self,builder,layer)
+            local trash = self.Trash
+            local landingAnimManip = CreateAnimator(self)
```
Comment (clyfordv) [inline units/UEA0107/UEA0107_Script.lua:49]: This needs to be assigned to `self.landingAnimManip`

File: units/UEB0202/UEB0202_script.lua [hunk 3]
```diff
@@ -10,48 +10,62 @@
...
         TAirFactoryUnit.StartArmsMoving(self)
-        if not self.ArmSlider1 then
-            self.ArmSlider1 = CreateSlider(self, 'Arm01')
-            self.Trash:Add(self.ArmSlider1)
+        local trash = self.Trash
+        local armSlider1 = self.ArmSlider1
+        local armSlider2 = self.ArmSlider2
+
+        if not armSlider1 then
+            armSlider1 = CreateSlider(self, 'Arm01')
```
Comment (clyfordv) [inline units/UEB0202/UEB0202_script.lua:28]: need to assign this to `self.armSlider1`

File: units/UEB0202/UEB0202_script.lua [hunk 4]
```diff
@@ -10,48 +10,62 @@
...
+            TrashBagAdd(trash,armSlider1)
         end
-        if not self.ArmSlider2 then
-            self.ArmSlider2 = CreateSlider(self, 'Arm02')
-            self.Trash:Add(self.ArmSlider2)
+        if not armSlider2 then
+            armSlider2 = CreateSlider(self, 'Arm02')
```
Comment (clyfordv) [inline units/UEB0202/UEB0202_script.lua:32]: assign this to `self.armSlider2`

File: units/UEB0203/UEB0203_script.lua [hunk 5]
```diff
@@ -10,59 +10,78 @@
...
         TSeaFactoryUnit.StartArmsMoving(self)
-        if not self.ArmSlider1 then
-            self.ArmSlider1 = CreateSlider(self, 'Right_Arm')
-            self.Trash:Add(self.ArmSlider1)
+        local trash = self.Trash
+        local armSlider1 = self.ArmSlider1
+        local armSlider2 = self.ArmSlider2
+
+        if not armSlider1 then
+            armSlider1 = CreateSlider(self, 'Right_Arm')
```
Comment (clyfordv) [inline units/UEB0203/UEB0203_script.lua:37]: need `self.armSlider1 = armSlider1`

File: units/UEB0203/UEB0203_script.lua [hunk 6]
```diff
@@ -10,59 +10,78 @@
...
+            TrashBadAdd(trash,armSlider1)
         end
-        if not self.ArmSlider2 then
-            self.ArmSlider2 = CreateSlider(self, 'Center_Arm')
-            self.Trash:Add(self.ArmSlider2)
+        if not armSlider2 then
+            armSlider2 = CreateSlider(self, 'Center_Arm')
```
Comment (clyfordv) [inline units/UEB0203/UEB0203_script.lua:41]: need `self.armSlider2 = armSlider2`

File: units/UEB0302/UEB0302_script.lua [hunk 7]
```diff
@@ -18,41 +23,50 @@ UEB0302 = ClassUnit(TAirFactoryUnit) {
...
-        if not self.ArmSlider1 then
-            self.ArmSlider1 = CreateSlider(self, 'Arm01')
-            self.Trash:Add(self.ArmSlider1)
+        local trash = self.Trash
+        local armSlider1 = self.ArmSlider1
+        local armSlider2 = self.ArmSlider2
+        local armSlider3 = self.ArmSlider3
+
+        if not armSlider1 then
+            armSlider1 = CreateSlider(self, 'Arm01')
```
Comment (clyfordv) [inline units/UEB0302/UEB0302_script.lua:32]: assign to self

File: units/UEB0302/UEB0302_script.lua [hunk 8]
```diff
@@ -18,41 +23,50 @@ UEB0302 = ClassUnit(TAirFactoryUnit) {
...
+            TrashBagAdd(trash, armSlider1)
         end
-        if not self.ArmSlider2 then
-            self.ArmSlider2 = CreateSlider(self, 'Arm02')
-            self.Trash:Add(self.ArmSlider2)
+        if not armSlider2 then
+            armSlider2 = CreateSlider(self, 'Arm02')
```
Comment (clyfordv) [inline units/UEB0302/UEB0302_script.lua:36]: assign to self

File: units/UEB0302/UEB0302_script.lua [hunk 9]
```diff
@@ -18,41 +23,50 @@ UEB0302 = ClassUnit(TAirFactoryUnit) {
...
+            TrashBagAdd(trash, armSlider2)
         end
-        if not self.ArmSlider3 then
-            self.ArmSlider3 = CreateSlider(self, 'Arm03')
-            self.Trash:Add(self.ArmSlider3)
+        if not armSlider3 then
+            armSlider3 = CreateSlider(self, 'Arm03')
```
Comment (clyfordv) [inline units/UEB0302/UEB0302_script.lua:40]: assign to self

File: units/UEB1101/UEB1101_script.lua [hunk 10]
```diff
@@ -9,10 +9,23 @@
 ----****************************************************************************
 local TEnergyCreationUnit = import("/lua/terranunits.lua").TEnergyCreationUnit
```
Comment (clyfordv) [inline units/UEB1101/UEB1101_script.lua:10]: Something in this unit crashes the sim when it spawns

**Review by clyfordv — COMMENTED** [review]
1. All the UEF factory build arms got busted up
2. Lot of `local trash = self.Trash` for single instances where `trash` is used, but's that code style more than performance impact

**Comment by MrRowey** [comment]
> impact

Feel free just to fix them as don't have the time to currently

**Pull request closed without merging**